products/zoningtaxlots/models/product/qa/qa_null.sql

```diff
@@ -0,0 +1,223 @@
+{{ config(
+    materialized = 'table'
+) }}
+
+WITH new_version AS (
+    SELECT * FROM {{ ref('int__zoningtaxlots') }}
+),
+
+prev_version AS (
+    SELECT * FROM {{ source('recipe_sources', 'dcp_zoningtaxlots') }}
+),
+
+
+
+newnull AS (
+    SELECT
```

Comment:
This section seems prime for maybe not a macro but just some jinja templating - iterate through a list of columns and union them. There might be some existing dbt function which does something like this, or you can write a for loop.

`dbt compile --select ...` is a very helpful command for this -> it spits out the "final" query that it's going to run